tests/test_sparse_gp.cc

```diff
@@ -70,17 +94,17 @@ TEST(test_sparse_gp, test_scales) {
   high_resolution_clock::time_point end = high_resolution_clock::now();
   auto direct_duration = duration_cast<microseconds>(end - start).count();
 
-  LeaveOneOut loo;
   UniformlySpacedInducingPoints strategy(100);
-  auto sparse = sparse_gp_from_covariance(covariance, strategy, loo, "sparse");
+  auto sparse =
+      sparse_gp_from_covariance(covariance, strategy, indexer, "sparse");
 
   start = high_resolution_clock::now();
   auto sparse_fit = sparse.fit(large_dataset);
   end = high_resolution_clock::now();
   auto sparse_duration = duration_cast<microseconds>(end - start).count();
 
   // Make sure the sparse version is a lot faster.
-  EXPECT_LT(sparse_duration, 0.2 * direct_duration);
+  EXPECT_LT(sparse_duration, 0.3 * direct_duration);
```

Comment:
It got a little slower?

Reply:
Only the PITC version got slower, full FITC end up the same (at least as far as I could tell)